### ISSUE TITLE
Add scrollbar in panels

### ISSUE THREAD

**Issue by radare** [radare/radare2#13619]
e scr.scrollbar=true only works in visual and graph for now. will be good to have it in other places

**Comment by Vane11ope** [comment]
I tried couple of implementations but I'm not sure if this fits to panels which already has too much stuff on canvas lol

**Comment by radare** [comment]
see that i have created the horizontal scrollbar, maybe good th ave p= or omething too

**Comment by radare** [comment]
any feedback here?

**Comment by NishantPuri99** [comment]
@radare is this perhaps still open?
If yes, how can I try it out?

**Comment by trufae** [comment]
Yes. This issue is still not solved. The way to test this is by enabling ‘e scr.scrollbar=true’ and then pressing v to enter panels mode. No scrollbar is shown in the frame border. In visual ( V command) works as expected

**Comment by NishantPuri99** [comment]
Okay. Trying this out then
